### PR TITLE
Konkong k8s Cookbook Operators

### PR DESCRIPTION
**Pull request by cbarria**
This adds the following to the cookbook for the Konkong cluster:
- rke
- metalLB (added lhn)
- nginx Ingress
- cert-manager
- multus on the bridge interface

### CODE REVIEW

File: konkong/rke/cluster.yml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  role:
+  - controlplane
+  - worker
+  - etcd
+  labels:
+    role: storage-node
+    jupyterlab: ok
+#- address: konkong04.ls.lsst.org
+#  hostname_override: konkong04
+#  user: rke
```
Comment (hreinking) [inline konkong/rke/cluster.yml:37]: Remember to use only 3 etcd nodes

**Review by hreinking — APPROVED** [review]
lgtm

File: konkong/rke/cluster.yml (hunk 1 above)
Comment (cbarria) [reply konkong/rke/cluster.yml:37]: yes, will be adjusted when all 5 nodes are ready (to only 3etcd, 5 is too overkill for a 5 node cluster)

Comment (jhoblitt) [reply konkong/rke/cluster.yml:37]: There's no reason **not** to use 5 etcd nodes for a 5 node cluster.

**Review by jhoblitt — APPROVED** [review]

**Pull request merged**